README.md

```diff
@@ -111,10 +111,11 @@ make check
   - Type `k9s` and press enter
   - Press `0` to select all namespaces
   - Wait for all pods to be in the `Running` state (look for the `STATUS` column)
-  - Use the arrow key to select `nsga-memory` then press the `l` key to view logs from the pod
+  - Use the arrow key to select `imdb` pod then press the `l` key to view logs from the pod
```

Comment:
there are no logs for this pod by default - either we could update the log level so that it logs successful requests, or update this to select the webv pod in the imdb namespace.

Reply:
updated to view webv logs for imdb

Reply:
you should look at the webv-heartbeat and webv-imdb logs - the first is .2 rps the second is 10 rps. Which leads into the FB logs nicely.